packages/bpk-component-floating-notification/README.md

```diff
@@ -0,0 +1,53 @@
+# bpk-component-floating-notification
+
+## Installation
+
+```sh
+npm install bpk-component-floating-notification --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkFloatingNotification from 'bpk-component-floating-notification';
+import BpkIconHeart from '../../packages/bpk-component-icon/sm/heart';
+
+export default () => (
+  <BpkFloatingNotification
+    ctaText="View"
+    type={TYPE.dark}
+    icon={BpkIconHeart}
+    onClick={() => {}}
+    text="Killer Combo saved to New York and Miami 🎉"
+  />
+);
+```
+
+## Props
+
+| Property       | PropType     | Required | Default Value |
+| -------------- | ------------ | -------- | ------------- |
+| animateOnEnter | bool         | false    | true          |
+| animateOnExit  | bool         | false    | true          |
+| className      | string       | false    | null          |
+| ctaText        | string       | false    | null          |
+| hideAfter      | number       | false    | 4000          |
+| icon           | ReactElement | false    | null          |
+| onClick        | func         | true     | null          |
+| text           | string       | true     | null          |
```

Comment:
For required props the default value we enter `-` instead of `null` as it never puts null in it